includes/polling/cisco-nac.inc.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+use LibreNMS\Util\IP;
+
+if (!get_dev_attrib($device, 'poll_cisco-nac', 'true')) {
```

Comment:
It would be better to add a discovery module to detect if nac should be polled or not.

Reply:
I didnt find yet a way to detect the NAC enable.